mailbox/store/src/main/java/org/apache/james/mailbox/store/event/EventFactory.java

```diff
@@ -174,6 +177,46 @@ public Mailbox getMailbox() {
         }
     }
 
+    public final class MailboxLookupRightUpdate extends MailboxListener.MailboxACLUpdated {
```

Comment:
What's that? Why don't you name it in a more generic way? Like "MailboxAclUpdatedEvent"

Reply:
Where's my mailboxId? (We should NOT have an event definition relying on something **mutable**!!!)